.github/workflows/semgrep.yml

```diff
@@ -0,0 +1,17 @@
+name: Lint
+on:
+  workflow_dispatch:
+  pull_request_target:
```

Comment:
I would put a comment here, explaining why we need this instead of the classic pull_target, and also
the caveat I put before which is you can't really modify this file and get the effect immediately.

Reply:
Added comment